azure-pipeline-update-swagger.yml

```diff
@@ -0,0 +1,90 @@
+resources:
+  repositories:
+  - repository: FhirCodegen
+    type: github
+    name: microsoft/fhir-codegen
+    endpoint: FirelyTeam
+    ref: dev
+
+variables:
+- group: Git credentials
+- name: SwaggerUiDirectory
+  value: $(Build.SourcesDirectory)/firely-docs-firely-server/_static/swagger/
+- name: OpenApiDirectoryName
+  value: OpenApiJson
+- name: SwaggerConfigFilename
+  value: swagger-config.json
+
+- name: FhirServerUrl
+  value: https://server.fire.ly/
+- name: OpenApiOutputDirectory
+  value: $(SwaggerUiDirectory)/$(OpenApiDirectoryName)
+- name: BranchNamePrefix
+  value: swagger_ui_update_
+- name: RemoteBranchName
+  value: $(BranchNamePrefix)$(Build.BuildNumber)
+
+trigger: none
+pr: none
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- checkout: self
+- checkout: FhirCodegen
+- task: DotNetCoreCLI@2
+  displayName: Generating OpenAPI files from the capability statement
+  inputs:
+    command: 'run'
+    projects: '$(Build.SourcesDirectory)/fhir-codegen/src/fhir-codegen-cli/fhir-codegen-cli.csproj'
+    arguments: '--fhir-server-url $(FhirServerUrl) --resolve-external false --language-options OpenApi|SingleResponses=true|Metadata=true|SchemaLevel=names|MultiFile=true --output-path $(OpenApiOutputDirectory) --language OpenApi'
+- task: PowerShell@2
+  displayName: Preparing SwaggerUI configuration file
+  inputs:
+    targetType: 'inline'
+    script: |
+      $urls = gci $(OpenApiOutputDirectory) -Exclude '$(SwaggerConfigFilename)' | %{
+          # The code below converts file names to human-readable resource names. Examples:
+          # OpenApi_R4_AdverseEvent.json =>  "Adverse Event"
+          # OpenApi_R4_Bundle.json =>  "Bundle"
+          # OpenApi_R4__Root.json => "System Operations" (exception)
+          $name = $_.Name -replace '(.*_)+' `
+          -replace '.json' `
+          -replace '^Root$', 'SystemOperations' `
+          -csplit '(?=[A-Z])' -ne '' -join ' '
+      
+          [PSCustomObject]@{
+              name = $name;
+              url = "./$(OpenApiDirectoryName)/$($_.Name)"
+          }
+        }
+      
+      $json = New-Object -Type PSCustomObject `
+        | Add-Member -name "urls" -value $urls -MemberType NoteProperty -PassThru `
+        | Add-Member -name "urls.primaryName" -value "System Operations" -MemberType NoteProperty  -PassThru
+        | ConvertTo-Json
+      
+      Write-Host $json
+
+      $json | Out-File $(OpenApiOutputDirectory)/$(SwaggerConfigFilename)
+- script: |
+    cd $(Build.SourcesDirectory)/firely-docs-firely-server
+
+    if [[ `git status --porcelain` ]]; then
+      git checkout -b $(RemoteBranchName)
+
+      git config --global user.email 'automated-build@azuredevops.com'
+      git config --global user.name 'Azure DevOps build'
+
+      git add .
+      git commit -m 'Updating public documentation'
+
+      git remote add origin2 https://$(git_token)@github.com/FirelyTeam/firely-docs-firely-server.git
+      git push --set-upstream origin2 '$(RemoteBranchName)'
+
```

Comment:
I found docs on creating a PR using `git request-pull` https://hackernoon.com/how-to-git-pr-from-the-command-line-a5b204a57ab1

Reply:
This command is meant for a different thing :(
https://stackoverflow.com/a/45721044/1523432